WebService/v1/Models/SimulationApiModel/SimulationRateLimits.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationRateLimits
+    {
+        [JsonProperty(PropertyName = "ConnectionsPerSecond", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Are any of these properties required? If so, we should have a validation method.

Reply:
These are not required properties